Client/Frontend/Widgets/SiteTableViewController.swift

```diff
@@ -93,8 +93,8 @@ class SiteTableViewController: UIViewController, UITableViewDelegate, UITableVie
 
         tableView.delegate = self
         tableView.dataSource = self
-        tableView.registerClass(TwoLineTableViewCell.self, forCellReuseIdentifier: CellIdentifier)
```

Comment:
Why delete the `TwoLineTableViewCell` registration? Does that mean we aren't using `TwoLineTableViewCell`s anywhere? If not, why create a separate `HistoryTableViewCell` rather than putting those changes directly into `TwoLineTableViewCell`?